src/components/interfaces/MOBILE_API.xml

```diff
@@ -3102,6 +3164,252 @@
+        </param>
+        <param name="heatedWindshieldAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Windshield.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+        <param name="heatedRearWindowAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Rear Window.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+        <param name="heatedMirrorsAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Mirrors.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+    </struct>
+
+    <struct name="EqualizerSettings">
+        <description>Defines the each Equalizer channel settings.</description>
+        <param name="channelId" type="Integer" minvalue="1" maxvalue="100" mandatory="true"></param>
+        <param name="channelName" type="String" mandatory="false" maxlength="50">
+           <description>read-only channel / frequency name (e.i. "Treble, Midrange, Bass" or "125 Hz")</description>
+        </param>
+        <param name="channelSetting" type="Integer" minvalue="0" maxvalue="100" mandatory="true"></param>
+    </struct>
+
+    <struct name="AudioControlData">
+        <param name="source" type="PrimaryAudioSource" mandatory="false">
+            <description>
+            In a getter response or a notification, it is the current primary audio source of the system.
+            In a setter request, it is the target audio source that the system shall switch to.
+            If the value is MOBILE_APP, the system shall switch to the mobile media app that issues the setter RPC.
+        </description>
+        </param>
+        <param name="keepContext" type="Boolean" mandatory="false">
+            <description>
+            This parameter shall not be present in any getter responses or notifications.
+            This parameter is optional in a setter request. The default value is false.
+            If it is false, the system not only changes the audio source but also brings the default application
+            or system UI associated with the audio source to foreground.
+            If it is true, the system changes the audio source, but still keeps the current application in foreground.
+            </description>
+        </param>
+        <param name="volume" type="Integer" mandatory="false" minvalue="0" maxvalue="100">
+            <description>Reflects the volume of audio, from 0%-100%.</description>
+        </param>
+        <param name="equalizerSettings" type="EqualizerSettings" minsize="1" maxsize="100" mandatory="false" array ="true">
```

Comment:
Unnecessary space in `array =true`.

Reply:
@jacobkeeler Fixed at commit 593f38dd0a9cfb336623c77fa160eb6f871f806e